### ISSUE TITLE
add a parameter to establish a vote cap within distribution

### ISSUE THREAD

**Issue by sgdias** [dakk/lisk-pool3#18]
some pool owners would be interested in offering a vote cap to benefit to attract vote from smaller accounts.

e.g.: vote cap is set to 200k, then if somebody votes with 300k it only counts 200k weight for the distribution calculation.

note: I'm not clear how but we also need factor the delegate vote capacity, what is fair distribution when vote from community exceeded the vote capacity ?

**Comment by dakk** [comment]
> some pool owners would be interested in offering a vote cap to benefit to attract vote from smaller accounts.
> 
> e.g.: vote cap is set to 200k, then if somebody votes with 300k it only counts 200k weight for the distribution calculation.

Do you know which delegates want this feature?


> 
> note: I'm not clear how but we also need factor the delegate vote capacity, what is fair distribution when vote from community exceeded the vote capacity ?

The solutions are 2:
- reward dilution: like now
- don't consider / pay votes received after the weight reach the max

**Comment by sgdias** [comment]
it's really a nice to have proposal, I don't think is so demanded feature. If as product owner you want discarding the proposal based on your product criteria is totally fine.

**Comment by dakk** [comment]
> it's really a nice to have proposal, I don't think is so demanded feature. If as product owner you want discarding the proposal based on your product criteria is totally fine.

No, that's totally a good idea; I will AFK for the next week, so after the 13th I'll take a look
